viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -647,6 +647,31 @@ export class Cognite3DViewer {
     this.controls.setState(this.getCameraPosition(), target);
   }
 
+  /**
+   * Attempts to load the camera settings from the settings stored for the
+   * provided model. See {@link https://docs.cognite.com/api/v1/#operation/get3DRevision}
+   * and {@link https://docs.cognite.com/api/v1/#operation/update3DRevisions} for
+   * information on how this settings is retrieved and stored. This setting can
+   * also be changed through the 3D models management interface in Cognite Fustion.
+   * @param model The model to load camera settings from.
+   * @returns True if a camera was stored for the provided model and applied to the
+   * current camera settings.
+   * @example
+   * ```js
+   * if (!viewer.loadCameraFromModel(model)) {
```

Comment:
Can you imagine a use case when a user wants to handle that in some different way? I'm just thinking maybe we should use that fallback internally by default, without forcing users to write all the same code every time. Like 

```js
  loadCameraFromModel(model: CogniteModelBase): void {
    const config = model.getCameraConfiguration();
    if (config) {
      this.controls.setState(config.position, config.target);
    } else {
      this.fitCameraToModel(model)
   }
  }
```

Reply:
I was thinking about that and I was a bit unsure if that would be better. I chose the current approach as it's more flexible and that it allows the user to handle this themselves, but I dont have any really good arguments against doing what you propose... Not really sure. Any input @christjt @j-bjorne ?

Reply:
Just to add an argument here, I think for rare cases when a user actually wants different behavior it's still can be achieved with model.getCameraConfiguration(); (as you did in examples before).

Reply:
I'm convinced, I'll make the change.